torch/fx/passes/splitter_base.py

```diff
@@ -39,6 +40,35 @@
 DEFAULT_SKIP_FUSION = False
 DEFAULT_ALLOW_NON_TENSOR = False
 
+# ENV var and constants for node tracker
+
+TRACKER_DUMP_PATH = "_fx_net_tracker"
+NODES_SUFFIX = "_nodes.txt"
+ALL_SUFFIX = "_all.txt"
+
+ENV_FX_NET_ACC_SPLITTER_TRACKER_MODE = "FX_NET_ACC_SPLITTER_TRACKER_MODE"
+ENV_FX_NET_ACC_SPLITTER_TRACKER_DUMP_PATH = "FX_NET_ACC_SPLITTER_TRACKER_DUMP_PATH"
+ENV_FX_NET_ACC_SPLITTER_TRACKER_TRACKED_NODES = (
+    "FX_NET_ACC_SPLITTER_TRACKER_TRACKED_NODES"
+)
+
+DUMP_PREFIX = os.environ.get(
+    ENV_FX_NET_ACC_SPLITTER_TRACKER_DUMP_PATH, TRACKER_DUMP_PATH
+)
+
+"""
+Different modes of the event tracker:
+"0": Tracker not enabled (by default)
+"1": Tracker enabled but no dumps. Information available by setting breakpoints and visually inspect in pdb.
+"2": Tracker enabled and dumps all events to DUMP_PREFIX_all.txt
+"3": In addition to events dump, track nodes specified by ENV_FX_NET_ACC_SPLITTER_TRACKER_TRACKED_NODES
+     recursively and dump to DUMP_PREFIX_nodex.txt
+"4": In addition to events dump, track all nodes with more than 1 event recursively and dump to DUMP_PREFIX_nodex.txt
+"""
+TRACKER_MODE: Literal["0", "1", "2", "3", "4"] = os.environ.get(
+    ENV_FX_NET_ACC_SPLITTER_TRACKER_MODE, "0"
+)  # type: ignore[assignment]
```

Comment:
Instead of doing it by an envvar, can we just dump it by default to tlparse using trace_structured?

Reply:
I can make the full flat event dump by default and always dump to the path specified by TORCH_TRACE. But I want a way for the user to specify the nodes they are interested in and dump those nodes recursively. So seems we can't get rid of all env vars anyway?

Reply:
I think I can make a default dump of all events flat via trace_structured, while preserving the local dump for a quick debug process. 
Later we can followup to visualize in fancier ways via tlparse.